docker/linux/android/Dockerfile

```diff
@@ -15,24 +15,24 @@
 ARG FROM_IMAGE
 FROM ${FROM_IMAGE:-cobalt-build-base}
 
-# TODO: b/309157124 - add --no-install-recommends.
 RUN apt update -qqy
```

Comment:
Combine `apt update` and `apt install` into a single RUN command, produces less layers.